.github/workflows/auto-format.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     # However, using a personal access token will cause events to be triggered.
     # We need that to ensure a status gets posted after the auto-format commit.
     # We also want to trigger tests if the auto-format made no changes.
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
```

Comment:
There is an issue with 3.1.0.

Ref https://github.com/actions/checkout/issues/956